lib/TvmClient.js

```diff
@@ -323,6 +330,37 @@ class TvmClient {
     return creds
   }
 
+  /**
+   * Revoke the presign credentials from the TVM
+   * @param {string} endpoint - TVM API endpoint
+   *
+   * @private
+   * @returns {Promise<object>} success response
+   */
+  async _revokePresignCredentials (endpoint) {
```

Comment:
this is reusing a lot from _getCredentialsFromTVM
Here is my suggestion: 
- rename _getCredentialsFromTVM to _requestTVM
- change all this to 
```
    return this._requestTVM(urlJoin(this.apiUrl, TvmClient.AzureRevokePresignEndpoint))
```